### ISSUE TITLE
let Generate (sine) create frequencies below 1Hz

### ISSUE THREAD

**Issue by hwkcgn** [audacity/audacity#4807]
### Your idea

to demonstrate e.g. simple AM (and other things) it would be extremely useful if Generate could create frequencies below 1 Hz. 
example: create one mono track at 440Hz, create a second one at 0.1Hz, mix and render to new track -> instant AM demo.
i use audacity a lot in teaching and just today found out about this limitation (or was it introduced recently)?

### Problem to be solved

limitations in the usefulness of audacity as a demonstration/teaching tool

**Comment by LWinterberg** [comment]
FWIW: You can use the "change speed" effect to slow down the 1hz wave even further.

**Comment by LWinterberg** [comment]
Right. I'm afraid your use case is a bit too niche for Audacity. Given that a simple workaround is available (NB: You can even automate it using [macros](https://manual.audacityteam.org/man/macros.html)), as well as plugins (eg [AudioQAM](https://plugins4free.com/plugin/3930/)), I don't think this will get changed.
